### PR TITLE
Docs-1571 Added notes

### PR DESCRIPTION
**Pull request by apigirl**
### What does this PR do?

Adds notes specific to the new Datadog for Government site.

- https://docs-staging.datadoghq.com/kaylyn/gov-notes/account_management/
- https://docs-staging.datadoghq.com/kaylyn/gov-notes/account_management/saml/
- https://docs-staging.datadoghq.com/kaylyn/gov-notes/account_management/users/
- https://docs-staging.datadoghq.com/kaylyn/gov-notes/agent/guide/private-link/
- https://docs-staging.datadoghq.com/kaylyn/gov-notes/integrations/amazon_xray/
- https://docs-staging.datadoghq.com/kaylyn/gov-notes/integrations/pagerduty/
- https://docs-staging.datadoghq.com/kaylyn/gov-notes/integrations/alibaba_cloud/
- https://docs-staging.datadoghq.com/kaylyn/gov-notes/integrations/amazon_web_services/?tab=automaticcloudformation#role-delegation
- https://docs-staging.datadoghq.com/kaylyn/gov-notes/logs/archives/?tab=awss3
- https://docs-staging.datadoghq.com/kaylyn/gov-notes/logs/archives/rehydrating?tab=awss3
- https://docs-staging.datadoghq.com/kaylyn/gov-notes/help/

### Dependencies

This PR is dependent on this integrations PR:

- https://github.com/DataDog/dogweb/pull/55413

And _SHOULD NOT_ be merged until the `pull_config_preview.yaml` file has been reverted back to normal.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/agent/guide/private-link.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ further_reading:
 Datadog exposes AWS PrivateLink endpoints in <b>us-east-1</b>.
 </div>
 
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">The PrivateLink integration with Datadog does not support the Datadog for Government site.</div>
```
Comment (ruthnaebeck) [inline content/en/agent/guide/private-link.md:18]:
```suggestion
<div class="alert alert-warning">The Datadog PrivateLink integration does not support the Datadog for Government site.</div>
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 2]
```diff
@@ -36,7 +36,7 @@
     contents:
 
     - action: integrations
-      branch: prod
+      branch: kaylyn/site
```
Comment (ruthnaebeck) [inline local/bin/py/build/configurations/pull_config_preview.yaml:39]:
```suggestion
      branch: prod
```
For change back before merge

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]

**Comment by apigirl** [comment]
also adding https://github.com/DataDog/documentation/pull/9525#discussion_r560427446

File: local/bin/py/build/configurations/pull_config_preview.yaml (hunk 2 above)
Comment (apigirl) [reply local/bin/py/build/configurations/pull_config_preview.yaml:39]: ha, good idea. I'll wait to accept this until right before we merge.

**Review by ruthnaebeck — APPROVED** [review]

File: content/en/agent/guide/private-link.md [hunk 3]
```diff
@@ -14,6 +14,10 @@ further_reading:
 Datadog exposes AWS PrivateLink endpoints in <b>us-east-1</b>.
 </div>
 
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">The Datadog PrivateLink integration does not support the Datadog for Government site.</div>
```
Comment (rayene290) [inline content/en/agent/guide/private-link.md:18]: nit: I don't think 'integration' is the right word. Probably best to just say 'The Datadog for Government site does not support Private Link' or something along those lines.

File: i18n/en.json [hunk 4]
```diff
@@ -48,15 +48,15 @@
   },
 
   "email_support_sentence": {
-    "other": "Contact the Datadog support team by email at <a href=\"mailto:support@datadoghq.com\">support@datadoghq.com</a>"
+    "other": "Contact the Datadog support team by email at <a href=\"mailto:support@datadoghq.com\">support@datadoghq.com</a> (all sites)"
```
Comment (rayene290) [inline i18n/en.json:51]: The email for GovCloud / us1-fed is support@ddog-gov.com

**Pull request merged**